apps/vetrina-test/packages.dhall

```diff
@@ -0,0 +1,111 @@
+{-
```

Comment:
I guess we can get rid of this mini guide

Reply:
Don't know if it matters that much. It also includes nice documentation if someone (new to spago and purescript) wonders what that is.

Reply:
Compromise: We can just remove this file :D